### PR TITLE
Emptyfirstrow fix

### PR DESCRIPTION
**Pull request by virtualramblas**
Pull request to merge the changes made to fix the issue #4

### CODE REVIEW

File: project/plugins.sbt [hunk 1]
```diff
@@ -1,15 +1,17 @@
...
+
+addSbtPlugin("com.eed3si9n" % "sbt-unidoc" % "0.3.3")
+
+addSbtPlugin("com.jsuereth" % "sbt-pgp" % "1.0.0")
+
+addSbtPlugin("com.typesafe" % "sbt-mima-plugin" % "0.1.9")
+
+addSbtPlugin("com.eed3si9n" % "sbt-assembly" % "0.14.3")
+
+addSbtPlugin("com.typesafe.sbteclipse" % "sbteclipse-plugin" % "5.1.0")
```
Comment (nightscape) [inline project/plugins.sbt]: I'd rather not include the sbteclipse-plugin in the project-local plugins. Anyone who's working with Eclipse can put that line into `~/.sbt/0.13/plugins/plugins.sbt`.

File: src/main/scala/com/crealytics/spark/excel/ExcelRelation.scala [hunk 2]
```diff
@@ -1,138 +1,161 @@
...
+      if(sheet.getRow(rowIndex) != null) {
+        isRowEmpty = false
+        headers = sheet.getRow(rowIndex).cellIterator().asScala.to[Vector]
+      }
+      rowIndex += 1
+    }
+    
+    return headers
+  }
+}
```
Comment (nightscape) [inline src/main/scala/com/crealytics/spark/excel/ExcelRelation.scala]: It seems that there's a lot of whitespace change here. Do you happen to be on Windows and have not set `autocrlf`?
```
git config --global core.autocrlf true
```

**Comment by virtualramblas** [comment]
Hi Martin, I am done with the requested changes.

Best Regards,
Guglielmo

File: project/plugins.sbt (hunk 1 above)
Comment (virtualramblas) [inline project/plugins.sbt]: Reverted back to original (no Sbt Eclipse plugin registration).

File: src/main/scala/com/crealytics/spark/excel/ExcelRelation.scala (hunk 2 above)
Comment (virtualramblas) [inline src/main/scala/com/crealytics/spark/excel/ExcelRelation.scala]: Requested change has been pulled to be merged.

**Comment by nightscape** [comment]
Hi Guglielmo,

the formatting looks better now, but the code can be shortened a lot in Scala.
I commited a change based on your fix:
https://github.com/crealytics/spark-excel/commit/8427d1ec713da87102fb109558426b0a49a951fe
Can you try that version on your example file and get back to me?

**Comment by virtualramblas** [comment]
Hi Martin,

I can confirm that after your change 8427d1e everything is working as expected: issue #4 has been fixed.

Best Regards,
Guglielmo

**Comment by nightscape** [comment]
Ok, great! I'll release a new version then :+1:

**Comment by nightscape** [comment]
Version 0.8.3 is released

**Pull request closed without merging**